client/blocks/simple-site-rename-form/dialog.jsx

```diff
@@ -89,11 +89,11 @@ class SiteRenamerConfirmationDialog extends PureComponent {
 					eventName="calypso_siterename_areyousure_view"
 					eventProperties={ { new_domain: newDomainName } }
 				/>
-				<h1>{ translate( "Let's Review" ) }</h1>
+				<h1>{ translate( "Let's review…" ) }</h1>
 				<p>
 					{ translate(
-						'You are about to change your site address. Once changed, ' +
-							'your previous site address will be unavailable for you or anyone else.'
+						"You're about to change your site address. Once you confirm the change, " +
+							'this site address will no longer be available for future use.'
```

Comment:
Minor: We're mixing `"` and `'` here.

Reply:
IIRC this is the only way to do this without the linter complaining and thus failing the build.

Reply:
TIL 👍

Reply:
Yah, just because of the `You're` 🤷‍♂️

Reply:
Oh right, didn't see the `You're` 🙈
👮‍♀️*move along, nothing to see here*